src/components/binding/binding.js

```diff
@@ -275,3 +283,16 @@ genotet.BindingView.prototype.updateTrackWithMapping_ = function(fileName) {
 
   this.panel.updateTracks();
 };
+
+/**
+ * Updates coordinate difference in header.
+ * @param {number} coordinateDifference Coordinate difference of binding
+ * tracks.
+ * @private
+ * */
+genotet.BindingView.prototype.updateCoordinateDifferenceInHearder_ =
+  function(coordinateDifference) {
+    this.container.find('.extra-info')
```

Comment:
header is under the control of view.js and any change to header content is preferred to go through view API.